templates/vue-demo-store/app.vue

```diff
@@ -111,6 +111,7 @@ if (languages.value?.elements.length && router.currentRoute.value.name) {
 
 onMounted(() => {
   refreshCart();
+  // this request is needed to show the wishlist heart on every category page without any limit / page
```

Comment:
so why do we need a pagination here if all wishlist items are already loaded?

Reply:
This is to show if a product is already on the wishlist.
Or in other words, to show the :heart: